internal/integration/integration_test.go

```diff
@@ -378,6 +378,41 @@ func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackWrongObjectSucceed
 	})
 }
 
+
+func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackIgnoreArgsSucceed() {
+	assert.NoError(suite.T(), chdir(suite.T(), suite.remoteRepo), "unable to chdir into our remote Git repo")
+	// Disable the `receive.fsckObjects option
+	require.NoError(suite.T(), exec.Command("git", "config", "receive.fsckObjects", "true").Run())
+	require.NoError(suite.T(), exec.Command("git", "config", "receive.fsck.missingEmail", "ignore").Run())
+	require.NoError(suite.T(), exec.Command("git", "config", "receive.fsck.badTagName", "ignore").Run())
+
+	assert.NoError(suite.T(), chdir(suite.T(), suite.localRepo), "unable to chdir into our local Git repo")
+
+	createBogusObjectAndPush(suite, func(suite *SpokesReceivePackTestSuite, err error, _ []byte) {
+		assert.NoError(
+			suite.T(),
+			err,
+			"unexpected error running the push with the custom spokes-receive-pack program; it should have succeeded since fsck args are ignored")
+	})
+}
+
+func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackMissingArgsFails() {
+	assert.NoError(suite.T(), chdir(suite.T(), suite.remoteRepo), "unable to chdir into our remote Git repo")
+	// Disable the `receive.fsckObjects option
```

Comment:
NIT: you either close the starting ` or remove it

Reply:
And the comment shouldn't be the opposite? You are not disabling the fsckObjects option, but enabling it?

Reply:
Yes you are correct. This is a left over comment from a test I copied from above and forgot to remove. Good catch.